config/locales/en.yml

```diff
@@ -956,6 +956,7 @@ en:
       transfers:
         dialog:
           title: Transfer Samples
+          description: "The following files were selected to be transferred:"
```

Comment:
**nitpick:** This should probably read `The following samples were selected to be transfered`